verifiers/internal/gha/npm.go

```diff
@@ -134,7 +134,7 @@ func (n *Npm) verifyProvenanceAttestationSignature() error {
 	return nil
 }
 
-func (n *Npm) verifyPublishAttestationSignature() error {
+func (n *Npm) verifyPublishAttestationSignature(sigstoreTUFClient utils.SigstoreTUFClient) error {
```

Comment:
Rather than change the function, could the `Npm` struct be initialized with a TUF client (or the npm verification metadata)?

Reply:
We could, but it doesn't save much space, in terms on code.

Reply:
Given there are multiple functions that take in a TUF client, I think it would be more idiomatic to have the `Npm` struct include the TUF client, though `getAttestationKey` would need to be a method on the struct. Not strongly opinionated on this.